optaplanner-docs/src/modules/ROOT/pages/use-cases-and-examples/vaccination-scheduling/vaccination-scheduling.adoc

```diff
@@ -0,0 +1,266 @@
+[id="chap-optaplanner-vaccination_{context}"]
+= OptaPlanner Vaccination Scheduler
+
+:OPTAPLANNER-COMM:
+//Variable for the community version conditional statements.
+//:OPTAPLANNER-ENT:
```

Comment:
let's rename this to just "productized" (to be in sync with "productized" that productization (marek team) is using to signal it's a product build (and some kie projects have a maven profiles that use that).

If it needs to be namespaced, we can use "optaplanner-productized" instead. But I strongly doubt it should be.

cc @rsynek